README.md

```diff
@@ -25,13 +25,22 @@ external repositories for Jsonnet:
 ```python
 http_archive(
     name = "io_bazel_rules_jsonnet",
+    # TODO: Update this to reflect a later release.
```

Comment:
NB: I'm publishing this here in advance. Some subsequent coordination is necessary.